x-pack/test/detection_engine_api_integration/utils.ts

```diff
@@ -809,16 +809,18 @@ export const getSimpleRuleOutputWithWebHookAction = (actionId: string): Partial<
 export const waitFor = async (
   functionToTest: () => Promise<boolean>,
   functionName: string,
-  maxTimeout: number = 20000,
-  timeoutWait: number = 10
+  maxTimeout: number = 800000,
+  timeoutWait: number = 250
```

Comment:
NOTE: Timeout is increased to work if there is a "mis fire" of a rule and it takes 5 minutes for it to run and product results for tests. This happens once every 1k times or so. Once every 25 ci runs or so.